packages/server/src/api/controllers/static/index.js

```diff
@@ -115,6 +114,10 @@ exports.getSignedUploadURL = async function (ctx) {
   // Determine type of datasource and generate signed URL
   let signedUrl
   let publicUrl
+  const AWS_REGION =
+    datasource?.config?.region?.length > 0
+      ? datasource?.config?.region
+      : "eu-west-1"
```

Comment:
I don't think we should be doing any defaulting, there's no real basis for it. If there is no region in the config we should probably throw an error. Maybe best to check with @aptkingston, was there ever not a region present on the s3 datasource config?

Reply:
I originally just actually copied the convention from the existing dynamo client, which was the other place that used the AWS SDK. I agree though in principle that if no region is specified then we should just error out instead.

Currently there's no validation forcing a region to be picked, so it's possible to have an undefined region. Even if we add validation now, there's the possibility that existing datasources are out there which are empty and are currently defaulting to `eu-west-1` and working, so it could technically be a breaking change to remove the default...

Maybe best to just keep the fallback for now, but also add validation to prevent empty regions going forward.

As an aside @melohagan - you can shorten this expression to just:
```js
const AWS_REGION = datasource?.config?.region ?? "eu-west-1"
```

Reply:
Nice thanks! Made the change suggested.